### DISCUSSION TITLE
G1GC OS Support

### DISCUSSION THREAD

**Discussion by zekronium** [oracle/graal#9427] [General]
Hi,

I remember tracking some pulls/tasks in the projects board in regards to G1GC landing on windows/osx, which I cant seem to find now.

Is there a more clear timeline when these features may appear and some key PRs to follow?

**Comment by mikehearn** [comment]
If you mean in SubstrateVM, unfortunately there's no specific work being done on that right now. @lewurm has looked at making it work on macOS but not Windows. If you mean on regular GraalVM then G1 should work already.

**Reply by zekronium** [reply]
Is there limited demand for it on substrate vm on other platforms
